verifiers/internal/gha/verifier.go

```diff
@@ -255,10 +255,13 @@ func (v *GHAVerifier) VerifyImage(ctx context.Context,
 		return nil, nil, err
 	}
 
-	// Parse any provenance target repository set using environment variable COSIGN_REPOSITORY
-	provenanceTargetRepository, err := ociremote.GetEnvTargetRepository()
-	if err != nil {
-		return nil, nil, err
+	var provenanceTargetRepository name.Repository
+	// Consume input for --provenance-repository when set
+	if *provenanceOpts.ExpectedProvenanceRepository != "" {
+		provenanceTargetRepository, err = name.NewRepository(*provenanceOpts.ExpectedProvenanceRepository)
```

Comment:
where is `provenanceTargetRepository` used?

Reply:
It is used here: https://github.com/slsa-framework/slsa-verifier/pull/736/files#diff-2b75bff58ca87cf24fae495867d0eb3a039b63b9505b885b705ea55720777cbcR271-R272

Reply:
this is verified and added to cosign registryClientOpts

Reply:
gotcha. Sorry the diffs confused me and could not find the variable. Thanks.